src/nvim/spell.c

```diff
@@ -137,7 +137,7 @@ slang_T *first_lang = NULL;
 char_u *int_wordlist = NULL;
 
 typedef struct wordcount_S {
-  uint16_t wc_count;                // nr of times word was seen
+  int wc_count;                     // nr of times word was seen
```

Comment:
?

Reply:
It's the simplest way to take care of the conversion warning in line 1899:

```c
if ((wc->wc_count += count) < (unsigned)count) {    // check for overflow
```

Reply:
maybe just `(wc->wc_count += (uint16_t)count)` ? No problem if that truncates the value, the if condition is checking for that as well.

Reply:
That will still give a conversion warning. Even if both variables are `uint16_t` the sum will still be promoted to `int`. I'll either need to change the type like I did or rewrite the if statement.